bee-message/src/payload/tagged_data/mod.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-2021 IOTA Stiftung
```

Comment:
Copyright dates

Reply:
I am doing a PR this week-end to change all dates at once